scripts/votca_help2doc.in

```diff
@@ -0,0 +1,143 @@
+#! /usr/bin/env python3
+
+# Copyright 2009-2021 The VOTCA Development Team (http://www.votca.org)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import itertools
+import subprocess
+from typing import Iterable, List, NamedTuple
+
+VERSION = '@PROJECT_VERSION@ #TOOLS_GIT_ID#'
+PROGTITLE = 'THE VOTCA::TOOLS help to doc converter'
+VOTCAHEADER = f"""
+==================================================
+========   VOTCA (http://www.votca.org)   ========
+==================================================
+
+{PROGTITLE}
+
+please submit bugs to bugs@votca.org 
+votca_help2doc, version {VERSION}
+"""
+
+AUTHORS = "Written and maintained by the VOTCA Development Team <devs@votca.org>"
+COPYRIGHT = """
+Copyright 2009-2011 The VOTCA Development Team (http://www.votca.org)
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this  file
+except in compliance with the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+Unless  required by applicable law or agreed to in writing, software distributed under the
+License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY  KIND,
+either  express  or  implied.   See  the  License  for  the  specific  language  governing
+permissions and limitations under the License.
+"""
+DESCRIPTION = f"""{VOTCAHEADER}
+Convert a Votca CLI's help message to RST format.
+Examples:
+votca_help2doc.in -n csg_map -o csg_map.rst
+"""
+
+
+class Data(NamedTuple):
+    """Data class containing the help's section."""
+    name: str
+    description: str
+    options: str
+    authors: str = AUTHORS
+    copyright: str = COPYRIGHT
+
+
+def parse_user_arguments() -> argparse.Namespace:
+    """Read the user arguments."""
+    parser = argparse.ArgumentParser("help2rst", usage=argparse.SUPPRESS,
+                                     description=DESCRIPTION, formatter_class=argparse.RawDescriptionHelpFormatter)
+    parser.add_argument("-n", "--name", required=True,
+                        help="Name of the tool to extract the help message")
```

Comment:
This assumes the executable is in the path, which isn't that case during the build.

Options:
1. add a search path option
2. cut path away from name
3. add option to explicitly name of the program. (Default --name).

I would prefer 3.

Reply:
@junghans  I didn't understand option 3) could you please give me an example? You mean like:
```
votca_help2doc --name /absolute/path/to/executable
```

Reply:
`votca_help2doc --exe path/to/csg_map --name csg_map`

With the full path after`--name`, the full path will also show up in the rest page.

Reply:
I just push a fix for that

Reply:
You implement option 2, but that is actually more elegant now that I think about it!